changelogs/fragments/6127-yarn-ignore-warnings.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - yarn - fixes bug where yarn module tasks would fail when warnings were emitted from Yarn. The yarn.list method wasn't filtering out warnings (https://github.com/ansible-collections/community.general/issues/6127).
```

Comment:
```suggestion
  - yarn - fixes bug where yarn module tasks would fail when warnings were emitted from Yarn. The ``yarn.list`` method was not filtering out warnings (https://github.com/ansible-collections/community.general/issues/6127).
```